scripts/fetch-readme.js

```diff
@@ -189,6 +189,11 @@ const readmes = [
     repo: 'contentauth/c2pa-min',
     path: 'README.md',
   },
+  {
+    dest: resolve(__dirname, '../static/sb-alg-list.json'),
```

Comment:
As noted - do NOT copy this material to the CAI site.

Reply:
The file is no longer in the PR (committed by mistake and removed).